ote/src/cljs/ote/theme/colors.cljs

```diff
@@ -2,6 +2,7 @@
 
 (def blue "#0048c2")
 (def blue-light "#0052dd")
+(def blue-dark "#0029B8")
```

Comment:
While this probably looks ok, I now see current colour codes for blue don't match to UI spec colour codes. 
Eg. #0048c2/rgb(0,102,204) is `blue-dark`.
Would be good to alighn these in a future PR. I think the other like gray/red/orange are already aligned.

Reply:
This color was given by Jussi, so I don't think the spec is up to date.

Reply:
Ok these shoudl be double checked with him. Previously it has been discussed that he too takes the codes from the spec, and so should we.